Git-Discard-Unstaged-Changes.md

```diff
@@ -0,0 +1,13 @@
+### Discard unstaged changes
+
+To discard unstaged file changes, one can use the 'git checkout' command to do so. 
```

Comment:
In Markdown, when you are using a command or code specific stuff, use the backticks to enclose those snippets.

> ~~git checkout~~`git checkout`